src/main/java/com/ltonetwork/client/core/transacton/Association.java

```diff
@@ -35,6 +36,13 @@ public Association(String party, int type) {
         this.hash = "";
     }
 
+    public Association(JsonObject json) {
+        super(json);
+        this.party = (String) json.get("party");
+        this.associationType = (int) json.get("associationType");
+        this.hash = (String) json.get("hash");
```

Comment:
The hash might be omitted, in which case this should be null.

Reply:
In case the json doesn't include `hash`, the `json.get("hash")` will return null.